sirius-pipeline/README.md

```diff
@@ -1,2 +1,41 @@
-# Sirius Pipeline P4 Behavior Models
-**TODO**
\ No newline at end of file
+# Sirius Pipeline
+
+## Build the environment
+```
+make docker
+```
+
+## Build pipeline
+```
+make clean
+bmv2/sirius_pipeline.bmv2/sirius_pipeline.json
```

Comment:
Shouldn't there be a backslash `\` joining these two lines? They are both make targets. I get an error trying to "execute" the second line.

Reply:
Fixed